examples/sveltekit-houdini/src/routes/+page.gql

```diff
@@ -0,0 +1,5 @@
+query ExampleQuery @live {
+	todoList(by: { id: "todolist_01GRHTKYNGGZCHSRTE9CP7T591" }) {
```

Comment:
I would fetch collection and you no longer need to hard code ID: 
```
query ExampleQuery @live {
	todoListCollection(first: 100) {
		edges {
			node {
				title
			}
		}
	}
}
```

Reply:
Thanks for the input! I wanted to expand the example to look more like the urql and relay ones so this file wasn't really meant to be a good example, just something I could use to confirm the behavior